### DISCUSSION TITLE
Try to modify HTTP_MB_LIMIT to increase file size limit but no luck

### DISCUSSION THREAD

**Discussion by smatmscare** [Budibase/budibase#10421] [Help] [answered]
According to https://docs.budibase.com/docs/hosting-settings
I added HTTP_MB_LIMIT=200 to my hosting.properties to increase the default 20M upload limit
I did "docker-compose --env-file hosting.properties up -d" to apply the change
but it seems not working.
I even tried to add "HTTP_MB_LIMIT: 200" to docker-compose.yaml, but that does not work either
anyone has luck changing the default file size limit before? really appreciate your help!

**Comment by melohagan** [comment]
Hi @smatmscare 

Have you tried putting that variable in your `.env` file? It's a hidden file located in the same folder as your docker-compose file.

**Comment by smatmscare** [comment]
thanks for the reply, I only have docker-compose.yaml; envoy.yaml and hosting.properties there, can not find hidden .env file.

**Reply by melohagan** [reply]
You could try adding a `.env` file with that setting and then trying a `docker-compose down` and `docker-compose up`

What version of Budibase are you running? I wouldn't expect to see anything related to envoy.

**Comment by smatmscare** [comment]
I upgraded Budibase to 2.6.2 and added .env, but seems not help, here is the .env file 

```yaml
# Use the main port in the builder for your self hosting URL, e.g. localhost:10000
MAIN_PORT=10000

# This section contains all secrets pertaining to the system
# These should be updated
API_ENCRYPTION_KEY=xxxxxxxxxxxxxxxxx
JWT_SECRET=xxxxxxxxxxxxxxxxxx
MINIO_ACCESS_KEY=xxxxxxxxxxxxxxx
MINIO_SECRET_KEY=xxxxxxxxxxxxxxx
COUCH_DB_PASSWORD=xxxxxxxxxxxxx
COUCH_DB_USER=xxxxxxxxxxxxxxx
REDIS_PASSWORD=xxxxxxxxxxxxxxx
INTERNAL_API_KEY=xxxxxxxxxxxxxxx
HTTP_MB_LIMIT=200

# This section contains variables that do not need to be altered under normal circumstances
APP_PORT=4002
WORKER_PORT=4003
MINIO_PORT=4004
COUCH_DB_PORT=4005
REDIS_PORT=6379
WATCHTOWER_PORT=6161
BUDIBASE_ENVIRONMENT=PRODUCTION
```

**Comment by harmkroog** [comment]
Hi ive created an Tutorial for an docker container with client_max_body_size and https for you
https://github.com/Budibase/budibase/discussions/10506

Best regards Bonkersdeluxe

**Comment by rthien** [comment]
Firstly, I am a Docker novice. Recently, a NFP (not for profit) organization got their hands on a Synology NAS. I tried to use the following commands to create a Budibase container:

docker run -d --name=budibase2 \
-p 10000:80 \
-e HTTP_MB_LIMIT=1000 \
-v /volume1/docker/budibase2/data:/data \
-v /volume1/docker/budibase2/db:/opt/couchdb/data \
--restart always \
budibase/budibase

With the above, I am still unable to upload attachments greater than 20MB.

I referred to the documentation: https://docs.budibase.com/docs/hosting-settings

Any help on this is greatly appreciated. The NFP requires to upload video files as part of their attachments in their forms.

**Reply by melohagan** [reply, marked as answer]
Hey @rthien @smatmscare 

There is more work needed around this: https://github.com/Budibase/budibase/issues/8372

The HTTP limit does not apply to file sizes.
